docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -16,18 +16,18 @@ To install all the prerequisites needed for the Substrate build environment, lik
 curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 
-As a part of `getsubstrate` script, we will also install [additional scripts](getting-started/using-the-substrate-scripts.md) which allow you to quickly create a new Substrate node template in your working folder. We will call this chain `contract-chain`:
+As a part of `getsubstrate` script, we will also install [additional scripts](getting-started/using-the-substrate-scripts.md) which allow you to quickly create a new Substrate node template in your working folder. We will call this chain `contracts-chain`:
 
 ```bash
 cd /my/working/folder/
-substrate-node-new contract-chain <YourName>
+substrate-node-new contracts-chain <YourName>
```

Comment:
This should be either clone `substrate-package` or clone `substrate-node-template` with `substrate-package` branch.

I think we should just tell people to use the package

Reply:
I see the value on this change but it adds a lot of new steps.

Reply:
`substrate-node-new` vs `git clone `https://github.com/substrate-developer-hub/substrate-node-template`

should be about equivalent.

Reply:
```bash
git clone https://github.com/substrate-developer-hub/substrate-package
```
### Initialize the git submodules.
``` bash
cd substrate-package/
git submodule update --init
```

### Rename your project.
``` bash
chmod +x ./substrate-node-template/substrate-node-rename.sh
./substrate-node-template/substrate-node-rename.sh constracts-chain <YourName>
```